### PR TITLE
Integrate Anlyser Data  as a DataSource for Google Data Studio

### PR DESCRIPTION
**Pull request by dkaraj**

### CODE REVIEW

File: sources/anlyser.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "ANLYSER_DATA",
+  "name": "Anlyser Data",
+  "categories": ["ANALYTICS"],
+  "organization": "ANLYSER",
```
Comment (pfrisella) [inline sources/anlyser.json:5]: This org doesn't exist. It also needs to be added. See https://github.com/googledatastudio/ds-data-registry#add-an-organization

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/anlyser.json (hunk 1 above)
Comment (dkaraj) [reply sources/anlyser.json:5]: I added it.

File: sources/anlyser.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "ANLYSER_DATA",
```
Comment (pfrisella) [inline sources/anlyser.json:2]: File name needs to match ID. Either update filename to anlyser_data.json or ID to ANLYSER.

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**